content/troubleshooting/common-android-issues.md

```diff
@@ -89,3 +89,44 @@ Upgrading to the latest version of Gradle in the `gradle-wrapper.properties` fil
 
 You can refer to the Official Documentation from Android Developer guides to learn more about the latest compatible version [here](https://developer.android.com/studio/releases/gradle-plugin#updating-gradle).
 
+
+### Could not find method firebaseAppDistribution() for arguments...
+
+###### Description
+When trying to publish to Firebase app Distribution using Gradle build fails with this error "Could not find method firebaseAppDistribution() for arguments..."
+
+###### Couse
+This issue are likely caused by missing Distribution Gradle plugin or missing dependency for the App Distribution Gradle plugin.
```

Comment:
When publishing to Firebase app Distribution using Gradle, build fails with this error "Could not find method firebaseAppDistribution() for arguments..."